cognite/seismic/protos/types.proto

```diff
@@ -136,6 +136,22 @@ message Geometry {
     }
 }
 
+/**
```

Comment:
Hmmm. I _think_ this moving-around should be harmless. But I'm not 100% sure. Could we test the SDK with this branch just to validate?

Reply:
The imports need to be updated in the cognite-seismic-sdk, I'll open a pr for it